brainstormforce/ultimate-addons-for-gutenberg
# Fix: Border Component - Console issue when border width does not have any units.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/3sAgbH1y7qY/43cd55f0-f123-4655-ab16-bfcee24df299?redirect=true&view=6e170ea7-8708-4546-8b82-23d2eebdf694&t=attributes

### Screenshots


### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop any block which has a border e.g. info-box CTA.
- Check style > border settings > Set width option and reset it. and check console issue is visible.
- Now move to the above branch and check issue is resolved or not.

### Checklist:
